compiler/test/dotty/tools/backend/jvm/DottyBytecodeTests.scala

```diff
@@ -118,6 +118,28 @@ class TestBCode extends DottyBytecodeTest {
     }
   }
 
+  @Test def switchOnStrings = {
+    val source =
+      """
+        |object Foo {
+        |  import scala.annotation.switch
+        |  def foo(s: String) = (s: @switch) match {
```

Comment:
Is the `@switch` required?  If not then I think it would be good to drop it so we assert that it still switches.

Reply:
It is not required and I've removed it from this test.